bottles/backend/managers/backup.py

```diff
@@ -134,14 +136,14 @@ def is_within_directory(directory, target):
 
                         return prefix == abs_directory
 
-                    def safe_extract(tar, path=".", members=None, *, numeric_owner=False):
+                    def safe_extract(_tar, _path=".", members=None, *, numeric_owner=False):
```

Comment:
Is there a reason why `_tar` and `_path` are preceding an underscore? This variable is used

Reply:
This is to distinguish it from the outer variable name, making the code more readable, which is also recommended by most Linters.
IIRC, we already chatted about this issue on Discord before, and Kinsteen has already did such changes in other PRs.